### PR TITLE
Addition gradebook table 

### PR DESCRIPTION
**Pull request by erikamats**
mapping student names for table rows - grades specific to student per assignment to come. As of now, the mapping is being done from the hard-coded data.

### CODE REVIEW

File: escuela/src/components/Pages/gradebook/Gradebook.js [hunk 1]
```diff
@@ -1,41 +1,62 @@
...
-    }
+      //close State
+    };
+    //close constructor
   }
 
-  viewCourses = () => {
-    this.setState({ allCourses: sampleCourses, students: [] })
+//added this which takes over onclick button method: viewStudents to test GradeTable
+  componentWillMount () {
```
Comment (Dissolutio) [inline escuela/src/components/Pages/gradebook/Gradebook.js:25]: All sample loading can be done in Component Did Mount, no need for setting state in two seperate lifecycle methods.

File: escuela/src/components/Pages/gradebook/Gradebook.js [hunk 2]
```diff
@@ -1,41 +1,62 @@
...
+    this.setState({
+      allCourses: sampleCourses,
+      allStudents: sampleStudents,
+      allGradebooks: sampleGradebooks,
+      allAssignments: sampleAssignments,
+    });
+// close componentDidMount
   }
 
+  viewCourses = () => {};
```
Comment (Dissolutio) [inline escuela/src/components/Pages/gradebook/Gradebook.js:38]: can remove empty function

File: escuela/src/components/Pages/gradebook/Gradebook.js [hunk 3]
```diff
@@ -1,41 +1,62 @@
...
+        <hr/>
+          <GradebookList
+            gradebooks={this.state.gradebooks}
+            assignments={sampleAssignments}
+          />
+          <button onClick={this.viewCourses}>View All Courses</button>
+          <button onClick={this.viewStudents}>View All Students</button>
+          <StudentsList students={this.state.students} />
+          <CourseList courses={this.state.courses} />
+        </div>
```
Comment (Dissolutio) [inline escuela/src/components/Pages/gradebook/Gradebook.js:57]: Not that big of a deal, but we are import sample data, setting it to state, and then passing some state and some sampleData as props, like in <GradebookList>, which is funny.

File: escuela/src/components/nav/NavigationBar.js [hunk 4]
```diff
@@ -27,6 +27,11 @@ export default () => {
               Home
 						</NavLink>
           </li>
+          <li className="nav-item" >
+            <NavLink to="/Assignment" className="nav-link" >
```
Comment (Dissolutio) [inline escuela/src/components/nav/NavigationBar.js:31]: URLs should be lowercased, `<Navlink to="/assignment" />`

**Review by Dissolutio — APPROVED** [review]
Great work, good anticipation of data (I'll deliver some soon I promise xD). The only suggestions I made are not critical.

File: escuela/src/components/nav/NavigationBar.js (hunk 4 above)
Comment (erikamats) [reply escuela/src/components/nav/NavigationBar.js:31]: fixed

File: escuela/src/components/Pages/gradebook/Gradebook.js (hunk 1 above)
Comment (erikamats) [reply escuela/src/components/Pages/gradebook/Gradebook.js:25]: Done!

File: escuela/src/components/Pages/gradebook/Gradebook.js (hunk 2 above)
Comment (erikamats) [reply escuela/src/components/Pages/gradebook/Gradebook.js:38]: Done - didnt want to remove it since it was there prior to

**Review by WebProFe — APPROVED** [review]

**Pull request merged**